src/newrelic/app_test.go

```diff
@@ -184,6 +184,54 @@ func TestConnectPayloadInternalHostname(t *testing.T) {
 	}
 }
 
+func TestConnectPayloadInternalDocker(t *testing.T) {
+	util := &utilization.Data{}
+	info := &AppInfo{}
+
+	// No docker ID, nil utilization data
+	info.DockerId = ""
+
+	b := info.ConnectPayloadInternal(1, nil)
+
+	result, _ := utilization.GetDockerId(b.Util)
+
+	if nil != b.Util {
+		t.Errorf("expected: %v\nactual: %v", nil, b.Util)
+	}
+
+	// No Docker ID, utilization data
```

Comment:
How is this test different that the one above?  Can't you check both test expectations after the first test runs?

Reply:
The above test passes `nil` for the `util` object, this test passes a non-`nil` `util` object.

Reply:
I'm not seeing how `util` is non-nil in the second test.

Reply:
I see how this code works now - I was overlooking the `nil` arg.